apps/crn-frontend/src/network/working-groups/WorkingGroupProfile.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  NotFoundPage,
+  WorkingGroupAbout,
+  WorkingGroupPage,
+} from '@asap-hub/react-components';
+import { network, useRouteParams } from '@asap-hub/routing';
+import { ComponentProps, FC } from 'react';
+import { useWorkingGroupById } from './state';
+
+const WorkingGroupProfile: FC = () => {
+  const route = network({}).workingGroups({}).workingGroup;
+  const { workingGroupId } = useRouteParams(route);
+  const workingGroup = useWorkingGroupById(workingGroupId);
+
+  if (workingGroup) {
+    const props: ComponentProps<typeof WorkingGroupPage> = {
+      id: workingGroup.id,
+      name: workingGroup.title,
+      members: [],
+      complete: false,
+      description: workingGroup.description,
+      externalLink: workingGroup.externalLink,
+      externalLinkText: workingGroup.externalLinkText,
+      lastUpdated: new Date(workingGroup.lastModifiedDate).toISOString(),
+    };
+
+    return (
+      <WorkingGroupPage {...props}>
+        <WorkingGroupAbout
+          members={props.members}
+          description={props.description}
+          pointOfContact={props.pointOfContact}
+        />
```

Comment:
I wonder if we don't need something like this

https://github.com/yldio/asap-hub/blob/master/apps/crn-frontend/src/network/groups/GroupProfile.tsx#L140


If we access https://2292.hub.asap.science/network/working-groups/37381e21-1e96-44fb-b55d-cd729a307e55

<img width="1012" alt="Screen Shot 2022-11-21 at 13 50 06" src="https://user-images.githubusercontent.com/16595804/203113069-24ff2df2-c520-4cb1-9f87-0391dc88188e.png">

The tab "About" appears as unselected, but if we access https://2292.hub.asap.science/network/groups/f4823066-418e-4bc4-9cda-0042e59cf47e and we wait, the url changes to `/about` and "About" tab appears as selected

Reply:
agree, although that would be part of the `WorkingGroupHeader.tsx` file and I think that bit is still WIP so it can be tackled later, for instance when more tabs are added, what do you think?

Reply:
I wonder if it isn't part of [apps/crn-frontend/src/network/working-groups/WorkingGroupProfile.tsx](https://github.com/yldio/asap-hub/pull/2292/files/76c6a487cf880f63d79cb8285fce1e3bc90a5b57#diff-e6e86287bdeab9dcfb3ac0c6241bb7b581e788f50bfdf8b3b2784c8209e985d9) because in the example I've sent it's part of `GroupProfile` page https://github.com/yldio/asap-hub/blob/master/apps/crn-frontend/src/network/groups/GroupProfile.tsx#L140